src/applications/disability-benefits/all-claims/validations.js

```diff
@@ -266,10 +273,12 @@ export const isInFuture = (err, fieldData) => {
  * @param {string} fieldData - The data associated with the current schema. Expected separation date
  */
 export const isLessThan180DaysInFuture = (errors, fieldData) => {
-  const enteredDate = moment(fieldData);
-  const in180Days = moment().add(180, 'days');
-  if (enteredDate.isValid()) {
-    if (enteredDate.isBefore()) {
+  const enteredDate = parseDate(fieldData);
+  if (enteredDate && enteredDate.isValid()) {
+    const today = parseDate(new Date().toISOString().split('T')[0]);
```

Comment:
I think you could use `getToday` from ['./utils/dates/dateHelper'](https://github.com/department-of-veterans-affairs/vets-website/blob/main/src/applications/disability-benefits/all-claims/tests/utils/dates/dateHelper.js#L24) here.